utilities/conv_gefs2datm/comp_f77_code.csh

```diff
@@ -0,0 +1,6 @@
+#! /bin/csh
+
+set name = $1
+#--- in hera 
+ifort -132 ${name}.f -o ${name} -L/apps/netcdf/4.6.1/intel/16.1.150/lib -lnetcdff
```

Comment:
the used netcdf library is different (4.6.) than the one loaded at conv_gefs2datm.fort.csh (netcdf 4.5.), please be concistent throughout the app.